src/xo-app/tasks/index.js

```diff
@@ -112,19 +141,24 @@ export default injectIntl(
     const { formatMessage } = intl
     return <Page header={HEADER} title={`(${nTasks}) ${formatMessage(messages.taskPage)}`}>
       <Container>
-        {map(pools, pool =>
-          <Row>
-            <Card>
-              <CardHeader key={pool.id}><Link to={`/pools/${pool.id}`}>{pool.name_label}</Link></CardHeader>
-              <CardBlock>
-                {map(pendingTasksByPool[pool.id], task =>
-                  <TaskItem key={task.id} task={task} />
-                )}
-              </CardBlock>
-            </Card>
-          </Row>
-        )}
+        <Row>
+          <SelectPool
+            multi
+            value={state.pools}
+            onChange={this.linkState('pools')}
+          /> <br />
```

Comment:
Remove the `<br />`.

You can add a bottom margin to the row if you wish: `className='m-b-1'`.